sdk/content-components/documentation/docs/api/getSASJobExecutionUrl.md

```diff
@@ -0,0 +1,28 @@
+---
+id: getSASJobExecutionUrl
+title: getSASJobExecutionUrl
+---
+
+```
+getSASJobExecutionUrl(item, url): Promise<string | undefined>
+```
+
+Given a target job definition item, such as from an `onSelect` event, and a target deployment URL, generate a SAS Job Execution URL that will open the job and allow it to be executed if it is a job with an HTML prompt. This URL can be opened in a new tab or window, or embedded in an iframe if allowed by the security settings on the deployment.
+
+## Arguments
+
+### `item: Item`
+
+The target job definition [Item](Item.md).
+
+### `url: string`
+
+Specify the URL of the SAS Viya server. This is the full context root, including the protocol, optional port, and host.
+
+## Return value
+
+`getSASJobExecutionUrl` will return a Promise that resolves to the URL representing the job in SAS Job Execution unless a problem is encountered, in which case it will resolve to `undefined`. Problems could include passing in an item that does not represent a job definition, not passing in a URL string, or unexcpected errors with the internal service calls needed to generate the URL. Consumers should handle the `undefined` case since service call failures are always a possibility.
```

Comment:
<-- Is "Promise" a term that our users will be familiar with? I am not familiar with this, but then I haven't done much coding in the last 15 years. -->

`getSASJobExecutionUrl` will return a Promise that resolves to the URL representing the job in SAS Job Execution. 
 If a problem is encountered, the Promise resolves to `undefined`. Problems could include passing in an item that does not represent a job definition, not passing in a URL string, or unexpected errors with the internal service calls that are needed to generate the URL. Consumers should handle the `undefined` case, because service call failures are always a possibility.